docs/content/configuration/csi_ug_config_create_snapshots.md

```diff
@@ -32,9 +31,9 @@ When creating volume snapshots, be sure to follow all of the snapshot configurat
     kubectl apply -f <filename>.yaml
     ```
 
-3.  Verify that the VolumeSnapshot was created.
+3.  Verify that the volume snapshot was created.
 
-    Run the kubectl describe volumesnapshot command.
+    Run the `kubectl describe volumesnapshot` command.
```

Comment:
```suggestion
    Run the `kubectl describe volumesnapshot demo-volumesnapshot` command.
```
to avoid describing irrelevant volumesnapshots (same for [other](https://github.com/IBM/ibm-block-csi-driver/blob/8d18ae5a6d95b15894c814e59a27c133aea06d8a/docs/content/configuration/csi_ug_config_create_replication.md) places)

Reply:
@oriyarde. I prefer not to specify the name of the volumesnapshot. That was an example and this is a more general statement.
What I can write instead, if you think it's better, is "Run the `kubectl describe volumesnapshot` command for the volumesnapshot that was created"
or "Run the `kubectl describe volumesnapshot <volumesnapshot-name>` command."
- similar to how we keep the `kubectl apply -f <filename>.yaml` command general. let me know what you want and I'll make the changes

Reply:
`kubectl describe volumesnapshot <volumesnapshot-name>` would be the best option (after my suggestion).
I thought we only use `<>` when there is more than 1 yaml in the page, or when we didn't specify a name.
I think all the other single-yaml pages use a specific name

Reply:
DONE - i've also updated other for consistency